iai_apartment/urdf/kitchen.xarco

```diff
@@ -1649,4 +1649,94 @@
 		</link>
 
 	</xacro:macro>
+
```

Comment:
One more request: the line number of the comment is incorrect, the correct line number is 1398.
The "island" link should be renamed to "island_counter_top" because otherwise I expected it to be the complete island mesh, not just the top, similarly to how a cabinet link is the complete cabinet.

Reply:
I think I address it in the new commit. Please check.

Reply:
Please rename the link called "island" into "island_counter_top".